.travis.yml

```diff
@@ -9,8 +9,7 @@ env:
     - CATKIN_WS=~/catkin_ws
     - CATKIN_WS_SRC=${CATKIN_WS}/src
   matrix:
-    - CI_ROS_DISTRO="indigo"
-    - CI_ROS_DISTRO="jade"
+    - CI_ROS_DISTRO="kinetic"
```

Comment:
Sorry. Replacing Indigo and Jade with Kinetic wasn't a good advice. Travis is stuck to a Trusty system, which doesn't support Kinetic. Please revert this. Using the compatibility header, the Kinetic branch should compile in Indigo and Jade too now.